mmhuman3d/data/datasets/pipelines/transforms.py

```diff
@@ -327,7 +327,7 @@ def __call__(self, results):
         if 'keypoints2d' in results:
             assert self.flip_pairs is not None
             width = results['img'][:, ::-1, :].shape[1]
-            keypoints2d = results['keypoints2d']
+            keypoints2d = results['keypoints2d'].copy()
```

Comment:
Should we do the same to `keypoints3d` and `smpl_body_pose`?

Reply:
added in 43ae21d